projects/core/src/product/occ/converters/occ-product-search-normalizer.ts

```diff
@@ -0,0 +1,26 @@
+import { ProductSearchPage } from '../../../occ/occ-models/occ.models';
+import { Converter } from '../../../util/converter.service';
+import { Injectable } from '@angular/core';
+import { ProductImageNormalizer } from './product-image-normalizer';
+
+@Injectable()
+export class OccProductSearchNormalizer
+  implements Converter<ProductSearchPage, ProductSearchPage> {
+  constructor(private productImageNormalizer: ProductImageNormalizer) {}
+
+  convert(
+    source: ProductSearchPage,
+    target: ProductSearchPage = {}
+  ): ProductSearchPage {
+    target = {
+      ...target,
+      ...source,
+    };
+    if (source.products) {
+      target.products = source.products.map(product =>
+        this.productImageNormalizer.convert(product)
```

Comment:
somehow I'd imagine this normalizer would only need to be configured as a provider, rather then being referenced explicitly.

Reply:
Ok, good point. Changed it to use ConverterService instead (because search result uses product entities, that has already normalization token (and logic) defined).